stock_indicators/indicators/common/quote.py

```diff
@@ -86,4 +85,6 @@ def use(cls, quotes: Iterable["Quote"], candle_part: CandlePart):
         return CsQuoteUtility.Use[Quote](CsList(Quote, quotes), candle_part.cs_value)
 
 
-Quote = generate_cs_inherited_class(_Quote, CsQuote, "Quote")
+class Quote(generate_cs_inherited_class(_Quote, CsQuote)):
+    """A base wrapper class for a single unit of historical quotes."""
+    pass
```

Comment:
See other comment about deprecation.

Reply:
Technically speaking, these is an _aggregate quote_.  A `tick` or normal `quote` is really a spot price containing bid/ask information; however, I've not been using the term _aggregate_ anywhere so wouldn't necessarily apply it here.  Just a thought.

Reply:
Oh,, thanks for letting me know.